templates/CRM/Report/Form/Tabs/Filters.tpl

```diff
@@ -8,27 +8,21 @@
  +--------------------------------------------------------------------+
 *}
 
-  <div id="report-tab-set-filters" class="civireport-criteria">
-    <table class="report-layout">
+<div id="report-tab-set-filters" class="civireport-criteria">
+  <div class="report-layout">
       {assign var="counter" value=1}
-      {foreach from=$filters item=table key=tableName}
```

Comment:
Minor thing: Because there's no more tableName, down on line 37/31 it doesn't like it.

Reply:
thanks - updated